data/gtk-style.css

```diff
@@ -536,7 +536,7 @@ ActivityCard {
     transition: background-size 150ms ease;
     background-size: 110%;
     animation: unset; }
-  ActivityCard.narrative:hover #topbox > box > overlay > box {
+  ActivityCard:hover.narrative #topbox > box > overlay > box {
```

Comment:
Same. Please remove this hunk from the commit. See https://git-scm.com/book/fa/v2/Git-Tools-Interactive-Staging